lib/perl/Genome/Model/Tools/Varscan/SomaticFilterWorkflow.pm

```diff
@@ -170,22 +182,16 @@ sub run_process_workflow {
     my $self = shift;
     my $input = shift;
 
-    my $w = Workflow::Operation->create(
+    my $w = Genome::WorkflowBuilder::Command->create(
         name => "Process Varscan SNVs",
-        operation_type => Workflow::OperationType::Command->get(
-            'Genome::Model::Tools::Varscan::ProcessSomatic')
+        command => 'Genome::Model::Tools::Varscan::ProcessSomatic',
     );
     $w->parallel_by('status_file');
     $self->set_lsf_queue($w);
-    $w->log_dir($self->outdir);
-    $self->validate_workflow($w);
 
-    # Launch workflow
-    $self->status_message("Launching process workflow now.");
-    my $result = Workflow::Simple::run_workflow_lsf(
-        $w,
+    my $result = $self->execute_workflow_for_operation($w, 'process', {
```

Comment:
Here as well, named arguments would be clearer.  And since this method was created in this PR, I don't think its outside the scope. :)

``` perl
use Params::Validate qw(validate_pos :types);

sub execute_workflow_for_operation {
    my $self = shift;
    my %p = validate_pos(@_, {
            operation => {isa => 'Genome::WorkflowBuilder::Detail::Operation'},
            operation_shortname => {type => SCALAR},
            inputs => {type => HASHREF},
        }
    );
    # ...
}
```